### PR TITLE
fix typos, change mrjs → MRjs

### PR DESCRIPTION
**Pull request by lobau**
README clean up

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
 ![The mrjs logo](https://docs.mrjs.io/static/mrjs_nopadding.svg)
  
-An extensible WebComponents library for the Spatial Web
+An extensible Web Components library for the Spatial Web
```
Comment (hanbollar) [inline README.md:3]: Components should be singular (--> Component)

File: README.md [hunk 2]
```diff
@@ -50,24 +50,24 @@ npm run server
 
 ### Documentation:
 
-Check [docs.mrjs.io](https://docs.mrjs.io) for the full documentation or our [repository](https://github.com/Volumetrics-io/documentation).
+Check [docs.mrjs.io](https://docs.mrjs.io) for the full documentation, or our [repository](https://github.com/Volumetrics-io/documentation).
```
Comment (hanbollar) [inline README.md:53]: We should actually just change this to :

Check [docs.mrjs.io](https://docs.mrjs.io) or our [repository](https://github.com/Volumetrics-io/documentation) for the full documentation.

File: README.md [hunk 3]
```diff
@@ -169,9 +168,9 @@ customElements.get('mr-spacecraft') || customElements.define('mr-spacecraft', Sp
 
 #### Systems
 
-A System contains logic that is applied to all entities that have a corresponding Component, using the data stored by the component. Unlike Entities & Components, Systems have no HTML representation and are implemented entirely in JS.
+A System contains logic that is applied to all entities that have a corresponding Component, using the data stored by the component. Unlike Entities & Components, Systems have no HTML representation and are implemented entirely in JavaScript.
 
-When a component is attached or detatched from an entity, it is added or removed from its System's registry of entities.
+When a component is attached or detached from an entity, it is added or removed from its System's registry of entities.
```
Comment (hanbollar) [inline README.md:173]: need to add a to after attached, so - `attached to` , since we have detached from

**Review by hanbollar — APPROVED** [review]
added some quick comments - thanks for making these changes!

File: README.md (hunk 1 above)
Comment (hanbollar) [reply README.md:3]: 🤔 or maybe not - it sounds wrong as plural though, but looking again maybe?

i'm just going to leave this as highlighted and let you both determine the final version you want for this one

Comment (lobau) [reply README.md:3]: "An extensible library of Web Components for the spatial web"?

Comment (hanbollar) [reply README.md:3]: yea that sounds better

**Pull request merged**